app/student/excel/index.php

```diff
@@ -17,13 +17,93 @@
 
 include '../../header.php';
 ?>
+<html>
```

Comment:
Please see the comment at the top of this file. Your app should begin at `excel/excel.php` and "live inside" that sandbox

@vjrene can you explain why that is necessary?

Reply:
Hey @rckjairath -- Our project structure was set up so that you wouldn't need to include any of the header files or external .css files that were already in the project. For example, with the excel folders, The index.php includes all the external headers/footers and css files, and also has a holder that would handle your part of the app. If you look at the index.php, it includes the accompanying excel.php, where your part of app would go.

We did it like this also to keep the URL's clean. When you navigate to the excel page in the browser, the URL should be "student/excel/" OR "instructor/excel".

Reply:
I think the instructor pages are as your request. In my next update, I will make student pages in the same way as well.